articles/virtual-desktop/faq.yml

```diff
@@ -171,3 +171,7 @@ sections:
           | Availability set (newly created by user) | Blob storage | Enabled with "No" as default |
           
           
+      - question: |
+         Should I use WDAC or AppLocker to control which applications and drivers are allowed to run on my Win 10 Devices?
```

Comment:
The style guide says you need to define an acronym the first time you use it. Can you please spell out "WDAC" so customers who aren't familiar with the acronym can understand what you're talking about? Also, can you check with Monica Rush or Curtis Love about whether "WDAC" is approved to use in customer documentation?

Reply:
updated the documentation to add the actual name for WDAC.